### ISSUE TITLE
A11Y: Read prices on add to cart button

### ISSUE THREAD

**Issue by steinsebastian** [SAP/spartacus#15653]
Currently the prices (Base Price, Selected Options, Total) are only read in Read-Mode. This should be changed by adding the price information to the "add to cart" button.

Provide aria-label for add to cart button that reads. "Add To Cart. Base Price: ###, Selected Options: ###, Total Price: ###". (Text has been discussed with Jutta).

The price should also be read on the overview page ("Continue to cart" button)
